packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.module.js

```diff
@@ -0,0 +1,21 @@
+import ngAtInternet from '@ovh-ux/ng-at-internet';
+import ngOvhHttp from '@ovh-ux/ng-ovh-http';
+
+import component from './user-identitiy-documents.component';
+import routing from './user-identity-documents.routes';
+
+const moduleName = 'UserAccountIdentityDocuments';
+
+angular
+  .module(moduleName, [
+    ngAtInternet,
+    ngOvhHttp,
+    'oui',
+    'pascalprecht.translate',
+    'ui.router',
+  ])
+  .component('accountUserIdentityDocuments', component)
```

Comment:
prefixing with app related scope could be good:
```suggestion
  .component('dedicatedAccountUserIdentityDocuments', component)
```